youtube_dlc/extractor/tiktok.py

```diff
@@ -133,6 +133,8 @@ class TikTokIE(TikTokBaseIE):
     def _real_extract(self, url):
         video_id = self._match_id(url)
 
+        # If we only call once, we get a 403 when downlaoding the video.
+        webpage = self._download_webpage(url, video_id, note='Downloading video webpage')
```

Comment:
```suggestion
        self._download_webpage(url, video_id)
```
This might actually be a cleaner approach

Reply:
Ya, we don't really need it; that's better.